src/brevitas/graph/gptq.py

```diff
@@ -64,40 +73,69 @@ def __init__(
         self.act_order = act_order
         # How many subblock to use during GPTQ for each layer
         self.num_blocks = num_blocks
+
+        # It is possible to speed-up the GPTQ if
+        # we know how the scale factors are split along the in_channel
+        self.in_channel_subslice = in_channel_subslice
+
         self.disable_quant_inference = DisableEnableQuantization()
         self.orig_forward = self.model.forward
         self.model.forward = self.catch_stopfwd
+        self.group_of_parallel_layers = group_of_parallel_layers
 
     def _is_module_supported(self, module):
-        if isinstance(module, qnn.QuantConv2d) and (module.groups == 1 or
-                                                    (module.groups == module.out_channels)):
-            return True
+        if isinstance(module, SUPPORTED_CONV_OP):
+            if (module.groups == 1 or (module.groups == module.out_channels)):
+                return True
+            else:
+                return False
         elif isinstance(module, qnn.QuantLinear):
             return True
         else:
             return False
 
     def __enter__(self):
+        # If the user specifies which layers to execute in parallel, we follow that structure
+        # Otherwise each supported layer will be executed standalone
+        if self.group_of_parallel_layers is not None:
```

Comment:
Can we do it differently, where we allow to pass in only some groups of layers, and we can infer that the others have to run by themselves?

Reply:
Yes I can work on that